### PR TITLE
Feature/test conflicts

### PR DESCRIPTION
**Pull request by CarmenDumitrescu**
This branch is intended to create conflicts.

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-					<p>Contact</p>
-				</li>
-			</ul>
-	</div>
-	<div>
-		<p>
-			Etiam finibus bibendum risus. Fusce rhoncus sollicitudin eros nec congue. Curabitur in diam vel ipsum eleifend ultricies a eget tortor. Nulla et ullamcorper ipsum. Praesent et imperdiet dui, cursus facilisis metus. Nam tempor volutpat pellentesque. Nulla facilisi. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. In bibendum tristique mattis.
+    <body>
+        <div class="header-wrapper">
+            <!- Please indent using 4 spaces instead of tabs ->
```
Comment (CarmenDumitrescu) [inline html/index.html]: This line should be deleted.

File: html/index.html [hunk 2]
```diff
@@ -1,30 +1,30 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Test Webpage</title>
+    <title>Test Webpage</title>
```
Comment (CarmenDumitrescu) [inline html/index.html:4]: The entire file should be indented using 4 spaces, not tabs, and have a trailing newline.

**Comment by CarmenDumitrescu** [comment]
@gabrieltirchila please fix the conflicts as mentioned in the comments.

**Review by gabrieltirchila — APPROVED** [review]

**Comment by CarmenDumitrescu** [comment]
Please fix the conflicts, then approve and merge the branch.

**Review by gabrieltirchila — APPROVED** [review]

**Pull request merged**